diff --git a/node_modules/react/cjs/react.development.js b/node_modules/react/cjs/react.development.js
index 1a1dde6..bd3dc74 100644
--- a/node_modules/react/cjs/react.development.js
+++ b/node_modules/react/cjs/react.development.js
@@ -157,7 +157,10 @@
     }
     function getOwner() {
       var dispatcher = ReactSharedInternals.A;
-      return null === dispatcher ? null : dispatcher.getOwner();
+      if (dispatcher && !dispatcher.getOwner) {
+        console.log({dispatcher})
+      }
+      return null === dispatcher || "function" !== typeof dispatcher.getOwner ? null : dispatcher.getOwner();
     }
     function hasValidRef(config) {
       if (hasOwnProperty.call(config, "ref")) {
